### PR TITLE
Prep for cephcsi v2.0.1 release

### PR DESCRIPTION
**Pull request by Madhu-1**
Updated Files to push new v2.0.1 release and also updated the yaml files to reflect the new image tag.

fixes: #855
fixes: #829

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
i will rebase this one, once https://github.com/ceph/ceph-csi/pull/870 is merged in  master

File: docs/ceph-csi-upgrade.md [hunk 1]
```diff
@@ -186,7 +193,7 @@ For each node:
   - The pod deletion causes the pods to be restarted and updated automatically
     on the node.
 
-we have successfully upgraded cephfs csi from v1.2.2 to v2.0.0
+we have successfully upgraded cephfs csi from v2.0.0 to v2.0.1
```
Comment (nixpanic) [inline docs/ceph-csi-upgrade.md]: Should "v1.2.2 to v2.0.1" not be mentioned as well?

Comment (Madhu-1) [reply docs/ceph-csi-upgrade.md]: this doc is for upgrading from v2.0.0 to v2.0.1

Comment (Madhu-1) [reply docs/ceph-csi-upgrade.md]: updated doc now

**Comment by Madhu-1** [comment]
merging this one as bot won't merge it as the PR author is not a bot.

**Pull request merged**